fpdcc/ccfp-asset-dashboard
# Allow for the bid quarter to not be required

## Overview

This allows for users to not have to pick anything for the bid quarter when creating/editing a phase.

The field in the model already allowed for this to be blank/null, so this just makes it optional in the form, and adjusts areas where the name is dependent on the bid quarter. In those cases, the quarter is omitted all together, so the name would be `preliminary_engineering - 2025 - new` as opposed to `preliminary_engineering - null - 2025 - new`. I can absolutely change this if we want to!

Closes #204
### Demo

![image](https://user-images.githubusercontent.com/114717958/236840270-0c3e87b0-5c0d-4604-ae83-6670b7c01a6f.png)
---
![image](https://user-images.githubusercontent.com/114717958/236842317-04bc0a41-3ffe-43aa-9b6f-a5c348b22172.png)


## Testing Instructions

* Create a phase and save it without having chosen a bid quarter
* Confirm that the phase has been created and that any display names do not include null or none
